swagger.json

```diff
@@ -20560,7 +20560,7 @@
             "in": "query",
             "description": "",
             "required": false,
-            "type": "string"
+            "type": "integer"
```

Comment:
should be a string, no?

You change all next types in the swagger (maybe a vi macro ran too much times)

Reply:
Hi,

yeah you're right, need to update the PR.

Reply:
Im super confused right now. Checked the diffs on Github and locally on my device and the diffs of Github are somewhat... broken... probably because of a rebase I did yesterday?
I even get like several "name" fields in the same json object which is obviously wrong and doesn't match my code...

Please also see the images. 
https://ibb.co/wdGKKbR
https://ibb.co/KK2x7fn

Any pointers what I'm doing wrong?

Reply:
Hey there, any idea of how I can help with this?

Reply:
Maybe close this PR, remove the branch and try again? I dunno what the problem could be to be honest.